README.md

```diff
@@ -1 +1,32 @@
-#XSLT REPO
+# XSLT XML to XML data transformation
+
+## Description: 
+
+This project provides several XSLT files which can be applied to XML files downloaded from EasyDB. The XSLT files transform the XML files to other, differently structured XML files that can be imported into ResearchSpace. There are XSLT files for the CIDOC CRM entities builtwork, person, place and group.
```

Comment:
To goal of the framework is to transform  the XML dump(s) from EasyDB to the target XML file for the data mapping into CIDOC-CRM. The four  XSLT schema(s) have been defined according to the main entities of our data-model: builtwork (E22_Man-Made_Object), person (E21_Person), place (E53_Place) and group (E74_Group).